build.gradle

```diff
@@ -54,3 +56,25 @@ dependencyManagement {
 tasks.named('test') {
 	useJUnitPlatform()
 }
+
+openApiGenerate {
+  generatorName = 'spring'
+  inputSpec = "$rootDir/../lynx-scanner/openapi.yaml".toString()
```

Comment:
This path doesn't exist in the current project. Use the guy submodule already configured

Reply:
Done